.github/ISSUE_TEMPLATE/request_example.yml

```diff
@@ -0,0 +1,33 @@
+name: Example Request
```

Comment:
i think this might have leaked out from your other PR ( https://github.com/tldraw/tldraw/pull/2597 ) :) let's remove